src/app/shared/components/Breadcrumbs/BreadcrumbsItem/BreadcrumbsItem.tsx

```diff
@@ -429,7 +429,7 @@ const BreadcrumbsItem = (props: BreadcrumbsItemProps): JSX.Element => {
         <div
           ref={drop}
           className={`flex ${props.isHiddenInList ? 'w-full' : 'max-w-fit'} ${
-            props.item.isFirstPath ? 'flex-shrink-0 p-1' : 'min-w-breadcrumb flex-1 py-1.5 px-3'
```

Comment:
Removing this causes the first item in the breadcrumb to be too close to the right caret icon when navigated to other folder
<img width="194" alt="Screenshot 2023-09-29 at 15 27 06" src="https://github.com/internxt/drive-web/assets/34165099/d9304669-83cf-47f2-b940-5a07f840f3bb">